Assets/_Project/Scripts/UserInterface/Interfaces/IPopup.cs

```diff
@@ -0,0 +1,14 @@
+using System.Collections;
+using System.Collections.Generic;
+using PolSl.UrbanHealthPath.UserInterface.Popups;
+using UnityEngine;
+
+namespace PolSl.UrbanHealthPath.UserInterface.Interfaces
+{
+    //It can later be transformed into an abstract class. 
```

Comment:
It doesn't lie but this comment should be gone ;)

Reply:
Done.